Sources/HipMobileUI/Pages/ExhibitDetailsPage.xaml.cs

```diff
@@ -65,14 +62,7 @@ protected override void OnBindingContextChanged()
         {
             base.OnBindingContextChanged();
 
-            audioToolbarButton = new HideableToolbarItem
-            {
-                Icon = "ic_headset_white",
-                Text = Strings.ExhibitDetailsPage_AudioToolbar,
-                Parent = this
-            };
-            audioToolbarButton.SetBinding(MenuItem.CommandProperty, "ShowAudioToolbarCommand");
-            audioToolbarButton.SetBinding(HideableToolbarItem.IsVisibleProperty, "AudioAvailable");
+            ToggleAudioBarVisibility();
```

Comment:
That function should probably be renamed to something like `UpdateAudioBarVisibility`, since it doesn't actually *toggle* the visibility.